vitrina/orgs/views.py

```diff
@@ -1,10 +1,51 @@
 from django.views.generic import ListView
+from django.views.generic import DetailView
 
 from vitrina.orgs.models import Organization
 
 
 class OrganizationListView(ListView):
     model = Organization
-    queryset = Organization.public.order_by('-created')
+    queryset = Organization.public.order_by('title')
     template_name = 'vitrina/orgs/list.html'
     paginate_by = 20
+
+
+class OrganizationDetailView(DetailView):
+    model = Organization
+    template_name = 'vitrina/orgs/detail.html'
+
+    def get_context_data(self, **kwargs):
+        context_data = super().get_context_data(**kwargs)
+        organization = context_data.get('organization')
+        extra_context_data = {
+            'members': organization.representative_set.all(),
+            'datasets': organization.dataset_set.all(),
+            'detail_active': True,
+            'members_active': False,
+            'datasets_active': False
```

Comment:
```suggestion
            'url_name': request.resolver_match.url_name,
```

Then in templates, you can simply check `if url_name == 'organization-detail'`. This way you will have less tings to care about. Also, `request` is already available in template, so probably there is no need to set this context variable at all.